dali/test/dali_plugin_manager_test.cc

```diff
@@ -16,28 +16,45 @@
 #include "dali/plugin/plugin_manager.h"
 #include "dali/test/dali_test_matching.h"
 #include "dali/test/dali_test_utils.h"
+#include "dali/pipeline/pipeline.h"
 
 static const std::string& DummyPluginLibPath() {
-    static const std::string plugin_lib = dali::CurrentExecutableDir() + "/libcustomdummyplugin.so";
-    return plugin_lib;
+  static const std::string plugin_lib =
+      dali::test::CurrentExecutableDir() + "/libcustomdummyplugin.so";
+  return plugin_lib;
 }
 
 namespace other_ns {
 
-template <typename ImgType>
-class DummyTest : public ::dali::GenericMatchingTest<ImgType> {
+class DummyTest : public ::dali::DALITest {
```

Comment:
Nitpick, the GenericMatchingTest tested both CPU and GPU variants of the op. 
Now we just invoke the CPU one. I just want to check if this is fine?